
# [CVE-2019-16224](https://github.com/TeamSeri0us/pocs/tree/master/lmdb/lmdb%20initialization%20vuln)

## Description

- `An issue was discovered in py-lmdb 0.97. For certain values of md_flags, mdb_node_add does not properly set up a memcpy destination, leading to an invalid write operation. NOTE: this outcome occurs when accessing a data.mdb file supplied by an attacker.`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 7.5

## Scores

- **Exploitability Score**:
  - 10.0
- **Impact Score**:
  - 6.4
- **Base Severity**:
  - HIGH

## Other Information

- **Publish Date**:
  - 2019-09-11 15:15:11
- **Vulnerability Status**:
  - Analyzed

## References

- **cve@mitre.org**: https://github.com/TeamSeri0us/pocs/tree/master/lmdb/lmdb%20initialization%20vuln
